attributes/default.rb

```diff
@@ -25,7 +25,7 @@
 
 default['php']['pecl'] = 'pecl'
 
-default['php']['version'] = '5.6.40'
+default['php']['version'] = '7.2.31'
```

Comment:
So, I thought about doing this, and held off b/c on Debian 9, it still installs PHP 5.6 (I know, I know).

But if you’re ok, I’m ok.

Reply:
Debian attributes set it to 7 though

Reply:
🤔 

Well then! Let’s roll with it.